receiver/saphanareceiver/metadata.yaml

```diff
@@ -0,0 +1,604 @@
+name: saphanareceiver
+resource_attributes:
+  host:
```

Comment:
To avoid clash with [host semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/host.md) I think we might use `saphana.host` here maybe, or move it back to metrics attributes. Alternatively `net.peer.name` (as is mentioned in [tracing database semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md)) might be used - though probably it's more fit for the metric attribute